docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -36,9 +36,19 @@ before or after calling Terraform.
 The `terraform` block supports the following arguments:
 
 - `source` (attribute): Specifies where to find Terraform configuration files. This parameter supports the exact same syntax as the
-  [module source](https://www.terraform.io/docs/modules/sources.html) parameter for Terraform `module` blocks, including
-  local file paths, Git URLs, and Git URLS with `ref` parameters. Terragrunt will download all the code in the repo
-  (i.e. the part before the double-slash `//`) so that relative paths work correctly between modules in that repo.
+  [module source](https://www.terraform.io/docs/modules/sources.html) parameter for Terraform `module` blocks **except
+  for the Terraform registry** (see below note), including local file paths, Git URLs, and Git URLS with `ref`
+  parameters. Terragrunt will download all the code in the repo (i.e. the part before the double-slash `//`) so that
+  relative paths work correctly between modules in that repo.
+    - NOTE: the `source` parameter can be configured to pull Terraform modules from any Terraform module registry using
+      the `tfr` protocol. The `tfr` protocol expects URLs to be provided in the format
+      `tfr://REGISTRY_HOST/MODULE_SOURCE?version=VERSION`. For example, to pull the `terraform-aws-modules/vpc/aws`
+      module from the public Terraform registry, you can use the following as the source parameter:
+      `tfr://registry.terraform.io/terraform-aws-modules/vpc/aws?version=3.3.0`. If you wish to access a private module
```

Comment:
NIT:

This seems a little hidden to document a potentially important feature.

Perhaps one of the quick start guides should be updated to use Registry URLs? One benefit of this is the user could start trying / learning Terragrunt without having to create their own modules.

Reply:
Took a stab at this: fdbf47f

Reply:
Nice 👍